modyn/config/schema/pipeline.py

```diff
@@ -101,43 +97,7 @@ class PresamplingConfig(BaseModel):
     force_required_target_size: bool = Field(False)
 
 
-class DownsamplingConfig(BaseModel):
-    """Config for the downsampling strategy of SelectionStrategy."""
-
-    class Config:
-        # depending on the strategy, additional fields may be required. Please refer to the respective strategy class
-        # for exact configurable fields.
-        extra = "allow"
-
-    strategy: Literal["Craig", "GradMatch", "GradNorm", "KcenterGreedy", "Loss", "No", "Submodular", "Uncertainty"] = (
-        Field(
-            description="Strategy used to downsample the data."
-            "Only the prefix, i.e. without `DownsamplingStrategy`, is needed."
-        )
-    )
-    sample_then_batch: bool = Field(
-        False,
-        description=(
-            "If True, the samples are first sampled and then batched and supplied to the training loop. If False, "
-            "the datapoints are first divided into batches and then sampled."
-        ),
-    )
-    ratio: int = Field(
-        description="Ratio post_sampling_size/pre_sampling_size. E.g. with 160 records and a ratio of 50 we keep 80.",
-        min=0,
-        max=100,
-    )
-    period: int | None = Field(
-        None,
-        description=(
-            "In multi-epoch training and sample_then_batch, how frequently the data is selected. "
-            "`1` selects every epoch. To select once per trigger, set this parameter to 0."
-        ),
-        min=0,
-    )
-
-
-class MultiDownsamplingConfig(BaseModel):
+class MultiDownsamplingConfig(ModynBaseModel):
```

Comment:
Should this go into downsampling config.py or stay here? It is a bit confusing this follows the same naming scheme as the stuff in the other file.

Reply:
Sure let me move it to the downsampling config file. Thanks for catching it